pkg/artifacts/signable.go

```diff
@@ -88,13 +88,6 @@ type StructuredSignable struct {
 
 func (oa *OCIArtifact) ExtractObjects(tr *v1beta1.TaskRun) []interface{} {
 	imageResourceNames := map[string]*image{}
-	if tr.Status.TaskSpec != nil && tr.Status.TaskSpec.Resources != nil {
-		for _, output := range tr.Status.TaskSpec.Resources.Outputs {
-			if output.Type == v1beta1.PipelineResourceTypeImage {
-				imageResourceNames[output.Name] = &image{}
-			}
-		}
-	}
```

Comment:
Is `tr.Status.TaskSpec.Resources.Outputs` v1alpha1?
Just feel a bit confused what v1alpha1 dependency we are removing here.

Reply:
So the logic mainly focuses on v1beta1.PipelineResourceTypeImage. When this is removed, there's no logic left within this for loop anymore